packages/@react-spectrum/menu/stories/MenuTrigger.stories.tsx

```diff
@@ -41,67 +87,303 @@ let withSection = [
+  )
+  .add(
+    'multiple default selected key (uncontrolled, static)',
+    () => render(
+      <Menu onAction={action('onAction')} selectionMode="multiple" defaultSelectedKeys={['2', '5']} disabledKeys={['1', '3']}>
+        <Section title="Section 1">
+          <Item key="1">
+            One
+          </Item>
+          <Item key="2">
+            Two
+          </Item>
+          <Item key="3">
+            Three
+          </Item>
+        </Section>
+        <Section title="Section 2">
+          <Item key="4">
+            Four
+          </Item>
+          <Item key="5">
+            Five
+          </Item>
+          <Item key="6">
+            Six
+          </Item>
+        </Section>
+      </Menu>
+    )
   )
   .add(
-    'single default selected key (uncontrolled)',
-    () => render({}, {defaultSelectedKeys: ['Kangaroo'], selectionMode: 'single'})
+    'multiple default selected key (uncontrolled, generative)',
+    () => render(null, {}, {defaultSelectedKeys: ['Kangaroo', 'Devon'], selectionMode: 'multiple'})
   )
   .add(
-    'multiple selected key (controlled)',
-    () => render({}, {selectedKeys: ['Kangaroo', 'Devon'], selectionMode: 'multiple'})
+    'Menu with autoFocus=true',
+    () => render(null, {}, {autoFocus: true})
   )
   .add(
-    'multiple default selected key (uncontrolled)',
-    () => render({}, {defaultSelectedKeys: ['Kangaroo', 'Devon'], selectionMode: 'multiple'})
+    'Menu with autoFocus="first"',
+    () => render(null, {}, {autoFocus: 'first'})
+  )
+  .add(
+    'Menu with autoFocus="last"',
+    () => render(null, {}, {autoFocus: 'last'})
+  )
+  .add(
+    'Menu with keyboard selection wrapping false',
+    () => render(null, {}, {shouldFocusWrap: false})
   )
   .add(
     'align="end"',
-    () => render({align: 'end'})
+    () => render(
+      <Menu onAction={action('onAction')}>
+        <Item>One</Item>
+        <Item>Two</Item>
+        <Item>Three</Item>
+      </Menu>
+    , {align: 'end'})
   )
   .add(
     'direction="top"',
-    () => render({direction: 'top'})
+    () => render(null, {direction: 'top'})
   )
   .add(
     'shouldFlip',
-    () => render({shouldFlip: true})
+    () => render(null, {shouldFlip: true})
   )
   .add(
     'isOpen',
-    () => render({isOpen: true})
+    () => render(null, {isOpen: true})
   )
   .add(
     'defaultOpen',
-    () => render({defaultOpen: true})
+    () => render(null, {defaultOpen: true})
   )
   .add(
     'disabled button',
-    () => render({isDisabled: true})
+    () => render(null, {isDisabled: true})
   )
   .add(
     'multiselect menu',
-    () => render({}, {selectionMode: 'multiple'})
+    () => render(null, {}, {selectionMode: 'multiple'})
   )
   .add(
     'no selection allowed menu',
-    () => render({}, {selectionMode: 'none'})
+    () => render(null, {}, {selectionMode: 'none'})
   )
   .add(
     'closeOnSelect=false',
-    () => render({closeOnSelect: false}, {})
+    () => render(null, {closeOnSelect: false}, {})
```

Comment:
while we're in the business of turning some of these stories for chromatic off, this one should probably not be run through chromatic